packages/base/src/keybindings.json

```diff
@@ -1,4 +1,14 @@
 [
+  {
+    "command": "jupytercad:undo",
+    "keys": ["Accel Z"],
+    "selector": ".jp-jupytercad-panel"
```

Comment:
```suggestion
    "selector": "body"
```

Reply:
Does that actually work? That's great if that works.

I believe the command should behave properly and only execute the command on the currently active JupyterCad editor, if there is one.

Reply:
> Does that actually work? That's great if that works.
> 
> I believe the command should behave properly and only execute the command on the currently active JupyterCad editor, if there is one.

Yes that works & as far as I had tested it, it behaves properly too.

Reply:
Even I had `body` as selector before 😆 in [`5518848`](https://github.com/jupytercad/JupyterCAD/pull/382/commits/55188485171439ea819ad37a36d87f494de306ca)
I saw https://github.com/QuantStack/jupytergis/pull/58 & thought that having fine grained control would be nicer, but definitely this is something we can standardize later too

Reply:
Sounds good to me :) Let's use the body selector for all commands for now then

Reply:
![image](https://github.com/user-attachments/assets/5cfd71e6-4ecc-4651-98d5-7b9d3f772328)
Verified again when editor is not enabled.

Reply:
> Let's use the body selector for all commands for now then

Done, Thanks for the instructions :)